echopype/echodata/echodata.py

```diff
@@ -607,7 +620,12 @@ def mapping_search_variable(mapping, keys, default=None):
                     "time1",
                     mapping_search_variable(
                         extra_platform_data,
-                        ["heave", "HEAVE", "vertical_offset", "VERTICAL_OFFSET"],
+                        [
+                            "heave",
+                            "HEAVE",
+                            "vertical_offset",
+                            "VERTICAL_OFFSET",
+                        ],
```

Comment:
This just must be because of `pre-commit run`? Nothing seems to be changed.

Reply:
Yepp pre-commit.